fe/src/main/java/org/apache/doris/planner/BrokerScanNode.java

```diff
@@ -225,62 +229,73 @@ private void initParams(ParamCreateContext context) throws AnalysisException, Us
      * @throws UserException
      */
     private void initColumns(ParamCreateContext context) throws UserException {
-        // This tuple descriptor is used for origin file
-        TupleDescriptor srcTupleDesc = analyzer.getDescTbl().createTupleDescriptor();
-        context.tupleDescriptor = srcTupleDesc;
-        Map<String, SlotDescriptor> slotDescByName = Maps.newHashMap();
-        context.slotDescByName = slotDescByName;
-
-        TBrokerScanRangeParams params = context.params;
-        // there are no columns transform
+        List<String> sourceFileColumns = context.fileGroup.getFileFieldNames();
```

Comment:
Why don't you analyze the `sourceFileColumns ` and `pathColumns ` in `DataDescription` ? The `originColumnNameToExprList ` is the finally columns expr including source file columns, path columns, and column expr.